### DISCUSSION TITLE
Supporting Cura 5 ???

### DISCUSSION THREAD

**Discussion by PhilBaz** [smartavionics/Cura#166] [General]
Hi Smart,

Just wondering if you will be moving onto/supporting Cura 5?

**Comment by smartavionics** [comment]
I don't think so because:

1 - my code base has diverged greatly from UM's code and so it would be a lot of work to port my features/fixes to 5.

2 - they have substantially changed how Cura is built and I don't think that it is compatible with the Raspberry Pi for which I produce 32 and 64 bit builds and I am not keen to drop those.

**Reply by PhilBaz** [reply]
Damn, I must say I'm sorry to hear that....your additions and changes have genuinely and meaningfully improved the usability of Cura for me.

**Comment by 5axes** [comment]
Same Type of question but in my case , Wondering If you you will not switching to Cura 5.0 but continue the development of your CURA Master version?

**Reply by smartavionics** [reply]
I will still be providing fixes for my builds but I am not currently working on any new features. If I get requests for features that I think are worthwhile and I have time to do the work, I would consider implementing them.

**Comment by CHSIVASANTHOSH** [comment]
Dear @smartavionics  greetings of the day , i would be glad if you can make detailed procedure/video  on how to build cura from source in windows system and how do we get our custom exe file of our own icon and naming so that on main window splash there won't be any ultimaker on it (untitled - Ultimaker cura)on the top left of the window with logo<--like this
